### ISSUE TITLE
Debugging NetStandard 2 assembly fails with "Unable to break execution 0x80004005"

### ISSUE THREAD

**Issue by itadapter** [dotnet/sdk#1901]
.Net Standard Assembly. Using .NET 4.7.1 launch project - a console app linking DYNAMICALY (via Assembly.load()) to the target DLL.  The breakpoints are not hit. The PAUSE does not work. VS 2017 15.5.5; tried running as admin; does not help.

An attempt to Pause the process:

![image](https://user-images.githubusercontent.com/2273574/35416036-b29bdb22-01f5-11e8-9df5-ecb6bb5a6321.png)

Anyone can shed some light?

**Comment by nguerrera** [comment]
@gregg-miskelly Do you know what might cause the debugger to error out with that message?

@itadapter Is this happening on multiple machines? Can you share a minimal repro project that causes this?

**Comment by gregg-miskelly** [comment]
If you go to the processes window (Debug->Windows->Processes) what does the 'Debugging' column say?
If you go to the modules window (Debug->Windows->Modules) is the modules window empty?

**Comment by itadapter** [comment]
Debugging processes: (trun.exe as expected)
![image](https://user-images.githubusercontent.com/2273574/35416938-261ac8ee-01f9-11e8-8ff4-3b5348f76a3f.png)

Debugging modules: EMPTY

**Comment by gregg-miskelly** [comment]
I believe the problem is that your debugger is configured to debug with .NET Core, but you actually want to debug .NET Full-framework.

What kind of project is your startup project? (Classic .NET Project or .NET Core), (Web/Console/WinForms)

**Comment by itadapter** [comment]
.NET 4.7.1   The assembly is .NET Standard 2.0 
And debugging works if you swithch roles - set the EXE as a startup target vs .DLL with EXE launcher

**Comment by nguerrera** [comment]
I see the repro now: 

* File -> New Project -> Class Library (.NET Standard)
* Project properties -> Debug:
  * Launch -> Executable
  * Executable -> Path to managed application

Something is deciding to use the CoreCLR debug engine here and I'm not sure how to override it. cc @BillHiebert

**Comment by itadapter** [comment]
@nguerrera yes it looks like it. Like I said, if you happen to have a LAUNCH EXE in the same assembly then set it as a startup (dynamically linking to .NET Std 2) then all works.

This is what I am doing now - creating a fake test container in the target solution to go around this problem

**Comment by itadapter** [comment]
@alelom that was 3 years ago, just moving to the next version of VS/fx fixed it, not sure what else changed
